bayesian/utility/db_gateway.py

```diff
@@ -98,6 +107,100 @@ def get_batch_ca_data(cls, ecosystem: str, packages) -> dict:
         logger.info("It took %s to fetch results from Gremlin.", elapsed_time)
         return response.json()
 
+    @classmethod
+    def get_vulnerabilities_for_packages(cls, ecosystem: str, packages) -> dict:
```

Comment:
There is a quite a bit of code duplicate between get_vulnerabilities_for_packages and get_package_details. Can you create a generic util function which takes just gremlin query and binding like below?

https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/src/utils.py#L325

Reply:
This is done.